Library/Homebrew/os/mac.rb

```diff
@@ -37,6 +37,15 @@ def outdated_release?
       version < "10.9"
     end
 
+    def os_name(version = @version)
```

Comment:
@MikeMcQuaid I think `name` is a bit too general to use as a method name 😄. Well to tell you the truth I just find it cumbersome to rewrite all the commit messages from "use os_name" to "use name".